test/adapters/adapters.spec.js

```diff
@@ -193,4 +250,37 @@ describe('adapters', () => {
         });
     });
 
+    _.each([
+        'mysql',
+        'postgres',
+        'sqlite'
+    ], (adapterName) => {
+        describe(`juttle-${adapterName}-adapter`, () => {
+            it('can write data and read back data using the juttle CLI', () => {
+                var id = uuid.v1().slice(0, 8);
+                var table = `test_${id}`;
+
+                // create a new table each time
```

Comment:
It probably doesn't but this means we have a cleaner setup if we were to execute a few more tests and do things to the data and would then need to introduce some cleanup (which in itself is horribly hard to correctly do). So the uniqueness provided by makes sure even rerunning the test within a few seconds of a previous run wouldn't give you any trouble.